src/adaptation/icap/History.cc

```diff
@@ -26,20 +26,20 @@ void Adaptation::Icap::History::start(const char *context)
     if (!concurrencyLevel++)
         currentStart = current_time;
 
-    debugs(93,4, HERE << "start " << context << " level=" << concurrencyLevel
+    debugs(93,4, "start " << context << " level=" << concurrencyLevel
```

Comment:
This PR automatically removes HERE, and nothing else should be required. The spacing issue is covered in another comment, so I will not repeat those arguments here.